src/github.com/stellar/horizon/db2/core/price_level.go

```diff
@@ -18,7 +18,7 @@ func (p *PriceLevel) PriceAsString() string {
 }
 
 // AmountAsString returns the amount as a string, formatted using
-// the amount.String() utility from go-stellar-base.
+// the amount.String() utility from go.
```

Comment:
It can be unclear for someone who is looking at the code for the first time. Someone may this there's `amount` package in standard go.